environment.yml

```diff
@@ -1,8 +1,5 @@
 # This file should have all the dependencies for development excluding the specific to the backends.
-name: ibis-dev
-channels:
-  - nodefaults
-  - conda-forge
```

Comment:
I guess with mambaforge and strict channel priority this is not needed in the CI. But I think it can still be helpful when devs create the environment locally with `conda env create`.